api/app/controllers/api/v1/event_organizers_controller.rb

```diff
@@ -0,0 +1,54 @@
+class Api::V1::EventOrganizersController < ApplicationController
+  before_action :set_event
+
+  def create
+    authorize @event, :manage_organizers?
+
+    user_id = organizer_params[:user_id]
```

Comment:
same issue as here: https://github.com/Project-Stage-Academy/Eventify-UA5070/pull/58/files#r2442406173 
(you can create new pr to fix that later, as this one is big)